packages/sfui/frameworks/react/components/SfListItem/types.ts

```diff
@@ -10,4 +10,5 @@ export interface SfListItemProps extends PropsWithChildren, PropsWithStyle {
   slotSuffix?: ReactNode;
   slotPrefix?: ReactNode;
   role?: string;
+  childrenTag?: ElementType<any> | undefined;
```

Comment:
By default `ElementType` sets `any` as an argument, so you don't have to pass it explicitly. Also, `?` makes property optional, so there's no need to set `| undefined`.
```suggestion
  childrenTag?: ElementType;
```

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2888/commits/63c0c62048b89164de41862f6a610a5acb1f2059